pallene/assignment_conversion.lua

```diff
@@ -0,0 +1,347 @@
+local util     = require "pallene.util"
```

Comment:
Don't forget to add the copyright notice

Reply:
Aha, My bad. I typically tend to forget things like that